src/glados/ASR/__init__.py

```
@@ -1,7 +1,48 @@
+        ValueError: If the specified engine type is not supported
+    """
+    if engine_type.lower() == "ctc":
+        from .ctc_asr import AudioTranscriber as CTCTranscriber
+
+        return CTCTranscriber()
+    elif engine_type.lower() == "tdt":
+        from .tdt_asr import AudioTranscriber as TDTTranscriber
+
+        return TDTTranscriber()
+    else:
+        raise ValueError(f"Unsupported ASR engine type: {engine_type}")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)